modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,98 @@ definition {
 		}
 	}
 
+	@description = "LPS-178783 - Verify that it's possible to send an e-mail notification with Action (Standalone)"
+	@priority = 5
+	test CanSendEmailViaActionStandalone {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given an Email Notification Template and Custom Object with an Action (Standalone),") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "Liferay",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 178783",
+				objectName = "CustomObject178783",
+				pluralLabelName = "Custom Objects 178783");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject178783");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject178783");
+		}
+
+		task ("When the timing is set on Notification Configuration And entry is triggered manually") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			Click(
+				key_itemName = "User Notifications",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+
+			Type(
+				locator1 = "ObjectAdmin#USER_NOTIFCATION_EVENT_CHECK_INTERVAL",
+				value1 = 1);
+
+			Type(
+				locator1 = "ObjectAdmin#USER_NOTIFCATION_EVENT_DAYS_LIMIT",
+				value1 = 1);
+
+			Button.clickSave();
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 178783");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.gotoAddObjectAction();
+
+			ObjectAdmin.fillObjectAction(
+				actionLabel = "Action Label",
+				actionName = "ActionName",
+				notificationName = "Notification Template",
+				thenAction = "Notification",
+				whenAction = "On After Add");
+
+			PortletEntry.save();
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject178783");
+
+			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry");
```

Comment:
Here, you should trigger the standalone action as the `When`

`When the Standalone Action is manually triggered`